.gitignore

```diff
@@ -26,3 +26,6 @@
 
 /.scannerwork/
 /sonar-project.properties
+
+application-local.properties
+src/main/resources/keystore.p12
```

Comment:
I put my `application-local.properties` at the root of my directory similar to a .env file.